src/core_plugins/console/public/index.html

```diff
@@ -12,11 +12,20 @@
           </a>
         </kbn-tooltip>
         <span dropdown>
-          <a class="editor_action" dropdown-toggle href="#">
-            <i class="fa fa-wrench"></i>
+          <a
+            id="consoleRequestOptions"
```

Comment:
I don't know if there's a formal plan for this but we're using camel case pretty much everywhere (e.g. test subject selectors, even CSS class names), so I think it makes sense to be consistent by applying camel case to IDs too.